common/scripted_effects/00_decisions_effects.txt

```diff
@@ -237,18 +234,15 @@ convert_whole_realm_to_feudalism_effect = {
 		}
 		custom_tooltip = feudal_liege_vassals_will_convert
 	}
-	##If I am Islamic, we will become clan.
+	##If I am an Orc, Dwarf or Tauren, we will become clan.
 	if = {
 		limit = {
 			is_independent_ruler = yes
-			
-			# Warcraft
-			always = no
-			# faith = { religion_tag = islam_religion }
+			uses_clan_government_trigger = yes
 		}
-		custom_tooltip = islamic_vassals_will_convert
+		custom_tooltip = odt_vassals_will_convert
```

Comment:
`odt_vassals_will_convert` should be localized. May do this in `Warcraft-Guardians-of-Azeroth-2\localization\english\wc_general_l_english.yml` or your own file.